docs/reports/unmapped.md

```diff
@@ -1,12 +1,12 @@
 # Mapping progress report
-| Ontology                           | Tot terms   | Tot excluded   | Tot deprecated   | Tot mappable _(!excluded, !deprecated)_   | Tot mapped _(mappable)_   | Tot unmapped _(mappable)_   | % unmapped _(mappable)_   |
-|:-----------------------------------|:------------|:---------------|:-----------------|:------------------------------------------|:--------------------------|:----------------------------|:--------------------------|
-| [ICD10WHO](./unmapped_icd10who.md) | 12,542      | 0              | 0                | 12,542                                    | 18                        | 12,524                      | 99.9%                     |
-| [ICD10CM](./unmapped_icd10cm.md)   | 95,847      | 15,452         | 0                | 80,395                                    | 1,160                     | 79,235                      | 98.6%                     |
-| [NCIT](./unmapped_ncit.md)         | 174,300     | 150,309        | 5,055            | 18,936                                    | 6,728                     | 12,208                      | 64.5%                     |
-| [OMIM](./unmapped_omim.md)         | 28,659      | 17,286         | 1,345            | 10,028                                    | 9,779                     | 249                         | 2.5%                      |
-| [ORDO](./unmapped_ordo.md)         | 9,448       | 292            | 0                | 9,156                                     | 8,934                     | 222                         | 2.4%                      |
-| [DOID](./unmapped_doid.md)         | 13,648      | 2,599          | 2,467            | 11,048                                    | 9,748                     | 1,300                       | 11.8%                     |
+| Ontology                           | Tot terms   | Tot excluded   | Tot deprecated   | Tot deprecated unmapped   | Tot mappable _(!excluded, !deprecated)_   | Tot mapped _(mappable)_   | Tot unmapped _(mappable)_   | % unmapped _(mappable)_   |
```

Comment:
I re-ran mapping progress docs and got these results:
| Ontology                           | Tot deprecated   | Tot deprecated unmapped |
|:-----------------------------------|:-----------------|:--------------------------|
| [ICD10WHO](./unmapped_icd10who.md) | 0                | 0                         |
| [ICD10CM](./unmapped_icd10cm.md)   | 0                | 0                         |
| [NCIT](./unmapped_ncit.md)         | 5,055            | 5,022                     |
| [OMIM](./unmapped_omim.md)         | 1,345            | 1,255                     |
| [ORDO](./unmapped_ordo.md)         | 0                | 0                         |
| [DOID](./unmapped_doid.md)         | 2,467            | 2,448                     |

Reply:
We have a high amount of terms that are deprecated/obsolete and mapped. I'm actually not sure though, is this number supposed to be low, high, or doesn't really matter? I'm guessing doesn't really matter.